src/test/ui/impl-trait/multiple-lifetimes/error-handling.stderr

```diff
@@ -3,10 +3,16 @@ error: lifetime may not live long enough
    |
 LL | fn foo<'a, 'b, 'c>(x: &'static i32, mut y: &'a i32) -> E<'b, 'c> {
    |        -- lifetime `'a` defined here                   ^^^^^^^^^ opaque type requires that `'a` must outlive `'static`
+   |
+   = help: Consider replacing `'a` with `'static`
```

Comment:
Use leading lowercase. We only use proper casing when we have long text with multiple sentences in a single help/note/warn.